docs/parser.md

```diff
@@ -3,4 +3,22 @@ title: "Parser"
 weight: 90
 ---
 
->This document is under construction.
\ No newline at end of file
+## Parser
+
+AsyncAPI Parser is a package used to parse and validate the AsyncAPI documents in your Node.js or browser application. These documents can be either in YAML or JSON format.
+
+Parser validates these documents using dedicated schema-supported plugins. These supported schemas are:
+
+- AsyncAPI schema (no plugin needed)
+- OpenAPI schema
+- JSON schema
+- Avro schema
+- RAML data-type schema
+
+> Note: If you use the parser as a package, you can also register external schemas. For example, you can also write your own schema.
```

Comment:
not needed in context of generator. Especially that there is no mechanism in place that would allow you to register some other parsers that are in place already